module-db/Interface/ContactRecord.cpp

```diff
@@ -1159,10 +1159,11 @@ auto ContactRecordInterface::buildNumberMatcher(unsigned int maxPageSize)
 {
     return utils::NumberHolderMatcher<std::vector, ContactNumberHolder>(
         [this](const utils::PhoneNumber &number, auto offset, auto limit) {
+            const auto numbers = !number.get().empty() ? contactDB->number.getLimitOffset(number.get(), offset, limit)
```

Comment:
is isValid our own method? We can consider fixing it....

Reply:
It seems not. It's using `IsValidNumber` from the libphonenumber, we only have a wrapper for that.